pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -366,6 +368,17 @@ func (c *Cluster) checkHealth(ctx context.Context) error {
 		}
 	}
 
+	// failover mons running on host path to use persistent volumes if VolumeClaimTemplate is set
```

Comment:
Let's also support converting in the other direction too if they have mons on PVCs and want to convert back to host path.

Reply:
update the changes to support failing over from host path to pv and vice versa.